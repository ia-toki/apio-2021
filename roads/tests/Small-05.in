c10234cabdfd6eae47773fb6f163e0350fc797e1
200
163 183 1000000000
199 49 1000000000
107 142 1000000000
51 181 1000000000
113 150 1000000000
177 97 1000000000
65 111 1000000000
193 16 1000000000
102 22 1000000000
91 180 1000000000
94 0 1000000000
68 193 1000000000
2 116 1000000000
189 180 1000000000
93 69 1000000000
167 188 1000000000
197 39 1000000000
138 2 1000000000
63 61 1000000000
170 192 1000000000
77 163 1000000000
44 21 1000000000
157 56 1000000000
7 138 1000000000
100 47 1000000000
58 145 1000000000
158 182 1000000000
129 180 1000000000
28 137 1000000000
67 127 1000000000
66 52 1000000000
23 82 1000000000
112 157 1000000000
147 129 1000000000
151 83 1000000000
43 74 1000000000
29 168 1000000000
140 31 1000000000
109 35 1000000000
171 21 1000000000
139 29 1000000000
18 80 1000000000
25 180 1000000000
0 76 1000000000
16 180 1000000000
160 161 1000000000
9 136 1000000000
49 70 1000000000
61 28 1000000000
39 64 1000000000
182 14 1000000000
150 7 1000000000
187 181 1000000000
75 168 1000000000
12 168 1000000000
180 116 1000000000
132 47 1000000000
119 159 1000000000
13 83 1000000000
86 115 1000000000
54 6 1000000000
186 116 1000000000
34 23 1000000000
144 77 1000000000
26 198 1000000000
83 127 1000000000
52 183 1000000000
166 163 1000000000
6 192 1000000000
79 83 1000000000
101 148 1000000000
149 159 1000000000
70 84 1000000000
74 181 1000000000
194 172 1000000000
152 114 1000000000
20 84 1000000000
32 72 1000000000
137 164 1000000000
173 76 1000000000
115 146 1000000000
104 103 1000000000
41 168 1000000000
179 146 1000000000
185 19 1000000000
111 189 1000000000
4 7 1000000000
120 11 1000000000
59 107 1000000000
40 85 1000000000
135 49 1000000000
98 183 1000000000
8 21 1000000000
133 193 1000000000
103 97 1000000000
159 142 1000000000
55 48 1000000000
155 142 1000000000
50 16 1000000000
45 125 1000000000
127 183 1000000000
184 113 1000000000
123 165 1000000000
162 7 1000000000
99 52 1000000000
146 181 1000000000
53 109 1000000000
96 158 1000000000
33 6 1000000000
14 138 1000000000
130 58 1000000000
35 49 1000000000
136 77 1000000000
188 82 1000000000
17 163 1000000000
87 55 1000000000
60 70 1000000000
90 115 1000000000
30 107 1000000000
36 160 1000000000
92 165 1000000000
198 137 1000000000
172 119 1000000000
72 84 1000000000
116 82 1000000000
56 37 1000000000
121 135 1000000000
125 183 1000000000
195 188 1000000000
174 61 1000000000
143 190 1000000000
71 20 1000000000
80 6 1000000000
176 147 1000000000
81 189 1000000000
85 153 1000000000
169 14 1000000000
114 175 1000000000
145 84 1000000000
15 63 1000000000
42 191 1000000000
117 103 1000000000
108 49 1000000000
128 18 1000000000
19 35 1000000000
38 125 1000000000
69 160 1000000000
168 142 1000000000
161 181 1000000000
89 49 1000000000
134 169 1000000000
156 16 1000000000
154 107 1000000000
165 55 1000000000
11 117 1000000000
62 98 1000000000
175 163 1000000000
192 35 1000000000
37 135 1000000000
64 105 1000000000
82 183 1000000000
76 47 1000000000
131 111 1000000000
88 129 1000000000
48 82 1000000000
110 97 1000000000
118 199 1000000000
97 107 1000000000
95 99 1000000000
78 189 1000000000
3 174 1000000000
46 154 1000000000
21 137 1000000000
164 163 1000000000
84 116 1000000000
1 12 1000000000
47 14 1000000000
142 183 1000000000
126 105 1000000000
181 116 1000000000
178 116 1000000000
124 50 1000000000
31 181 1000000000
191 55 1000000000
73 46 1000000000
24 145 1000000000
141 115 1000000000
196 2 1000000000
122 25 1000000000
106 107 1000000000
27 47 1000000000
5 90 1000000000
190 142 1000000000
153 105 1000000000
57 67 1000000000
22 183 1000000000
10 183 1000000000
105 52 1000000000
148 16 1000000000
