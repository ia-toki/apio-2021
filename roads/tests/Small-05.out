064bde9ff69ddc34b3b45c2d26d58873d85290d3
OK
199000000000 117000000000 63000000000 36000000000 18000000000 9000000000 4000000000 2000000000 1000000000 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
